internal/assert/assert.go

```diff
@@ -55,6 +55,9 @@ func (a Assert) Equal(want, got interface{}) {
 }
 
 func unmarshalAny(a *any.Any) proto.Message {
+	if a == nil {
+		return nil
+	}
```

Comment:
What happened here? Does this get a nil that it didn't get before?

Reply:
This one was weird, after upgrading the deps, some tests started to fail. The error was the comparison was nil when checking. 

Maybe if you could put a set of eyes on it and see if something else jumps out at you.

Reply:
I think that we just got lucky before. Any comparison involving an `any.Any` could have triggered this, since it makes `ptypes.Empty` return an error. Looks like ` RetryPolicyTypedConfig *any.Any` was added to a few types, which was probably one trigger.